### ISSUE TITLE
build may fail on Windows 10

### ISSUE THREAD

**Issue by felipenoris** [felipenoris/Mongoc.jl#44]
`Mongoc` package build fails on Windows 10, because libmongoc can't be dlopened.

```
Mongoc\deps\usr\bin\libmongoc-1.0.dll cannot be dlopen'ed
```

**Comment by felipenoris** [comment]
There's a possibility to change the builder to use mingw instead of Visual Studio, but I got stuck in https://jira.mongodb.org/browse/CDRIVER-3082 .

**Comment by felipenoris** [comment]
Also opened this issue regarding the use of Visual Studio and Appveyor to build binaries: https://jira.mongodb.org/browse/CDRIVER-3098.

**Comment by felipenoris** [comment]
Today I was able to compile libmongoc on a Windows 10 machine. On the same machine, I couldn't dlopen the library. So I guess this is an upstream issue.

**Comment by vilim** [comment]
I installed Mongoc without any issues now on Windows 10

**Comment by felipenoris** [comment]
@vilim good to hear that! Which version of Julia are you using? Can you post the result of `versioninfo()` here?

**Comment by vilim** [comment]
This is the output

```
Julia Version 1.3.0
Commit 46ce4d7933 (2019-11-26 06:09 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Xeon(R) CPU E5-2687W v3 @ 3.10GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)
Environment:
  JULIA_NUM_THREADS = 20
```

**Comment by rapus95** [comment]
How about building onto the artifact system? I.e. request a build from https://github.com/JuliaPackaging/Yggdrasil and use it as a jll package.

**Comment by waqasahmad531** [comment]
> I installed Mongoc without any issues now on Windows 10

Can someone please share the complete procedure? 

I have tried to install it few months ago but after many futile attempts  I decided to delay learning Julia. But, now I am trying again and still no luck.

**Comment by aviks** [comment]
Might be fixed by #67

**Comment by felipenoris** [comment]
Should be fixed by #67 now.
